src/main/java/nextstep/subway/line/domain/Line.java

```diff
@@ -58,11 +63,20 @@ public String getColor() {
         return sections;
     }
 
+    public BigDecimal getExtraFee() {
+        return extraFee;
+    }
+
     public List<Station> getStations() {
         LineSectionExplorer lineSectionExplorer = new LineSectionExplorer(sections);
         return lineSectionExplorer.getStations();
     }
 
+    public boolean isBelongedStation(Long stationId) {
+        List<Long> stationIds = getStations().stream().map(Station::getId).collect(Collectors.toList());
```

Comment:
```suggestion
        List<Long> stationIds = getStations().stream()
        .map(Station::getId)
        .collect(Collectors.toList());
```

행위 단위로 내려쓰는것이 가독성 및 디버깅에 유리한점이 많은것 같아요.

Reply:
으... 죄송합니다 급하게 막 쓰면서 놓친 부분이 되게 많네요 ㅜ 수정 진행하겠습니다.